src/main/webapp/i18n/en/exerciseAssessmentDashboard.json

```diff
@@ -10,29 +10,47 @@
             "pageHeader": "Exercise Assessment Dashboard for:",
             "testRunPageHeader": "Test Run Exercise Dashboard for exercise:",
             "yourStatus": "Your status",
+            "tutorParticipationStatus": {
+                "title": {
+                    "NOT_PARTICIPATED": "Not participated",
+                    "REVIEWED_INSTRUCTIONS": "Read the instructions",
+                    "TRAINED": "Trained on example submissions",
+                    "COMPLETED": "Completed assessing students submissions"
+                },
+                "description": {
+                    "NOT_PARTICIPATED": "Please read the assessment instructions carefully and confirm that you have understood them",
+                    "REVIEWED_INSTRUCTIONS": "Please open Example Submissions and start training",
+                    "TRAINED": "Now you can start assessing students submissions",
```

Comment:
Shouldn't it be students' submission?
And I would add a point at the end of the line (same for German)

Reply:
done